MarsFramework/Test/TestStarts.cs

```diff
@@ -23,10 +23,16 @@ public void Test_AddShareSkills()
                 ShareSkill page = new ShareSkill();
                 page.AddShareSkill(Driver);
```

Comment:
you should avoid passing the driver around, if you need it then create a variable in the page class and then use is inside the method, please fix this where ever you are passing the driver

Reply:
Thanks and fixed it.